### ISSUE TITLE
A11y_Azure Tools for VS Code [BM]_Create New Project_Keyboard: Unable to perform "Create New Project" and "Create New Function" using keyboard instead getting error message

### ISSUE THREAD

**Issue by Ashwinipola26** [microsoft/vscode-azurefunctions#2694]
### GitHubTags:
#Azure Tools for VS Code;#A11ySev1;#A11yTCS;#BM-Azure Tools for VS Code-MAC-Feb2021;#DesktopApp;#WCAG2.1.1;#MAC; #Keyboard#AINotInScope

### OS Details:
macOS Catalina Version 10.15.7

### Environment Details:
Application Name: Azure Tools for VS Code
Version: 1.53.2
Commit: 622cb03f7e070a9670c94bae1a45d78d7181fbd4
Date: 2021-02-11T11:45:54.515Z(6 days ago)
Electron: 11.2.1
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Darwin X64 19.6.0
Azure Function Version: 1.2.1

### Repro Steps:

1. Open Visual studio code
2. Tab till extension button and press enter on it.
3. Search for "Azure Functions" and select it.
4. Tab till "Install" button and press enter on it.
5. Tab till "Azure" button from left side pane and press enter on it.
6. Default screen will get open.
7. Tab till Create a Project and Press enter
8. Check whether user is able to Create New Project and Create Function using keyboard with out any error message

### Actual Result:
Unable to access "Create New Project" and "Create New Function" using keyboard instead getting error message

### Expected Result:
User should be able to perform "Create New Project" and "Create New Function" using keyboard without getting error message

### User Impact:
Keyboard users will not be able to Create New project and Create a Function using Keyboard

### Attachment for Reference:

[Bug_2694_Unable to perform "Create New Project" and "Create New Function" using keyboard instead getting error message.zip](https://github.com/microsoft/vscode-azurefunctions/files/6009144/Bug_2694_Unable.to.perform.Create.New.Project.and.Create.New.Function.using.keyboard.instead.getting.error.message.zip)

**Comment by ejizba** [comment]
This is caused by https://github.com/microsoft/vscode/issues/119369

**Comment by ejizba** [comment]
This is fixed for me in the latest insiders version of VS Code
